app/server/models/user.py

```diff
@@ -573,6 +574,16 @@ def refine_with_default_transfer_usages(self, most_common, nr_of_wanted):
                 break
         return most_common
 
+    def get_txn_limits(self, credit_transfer=None):
+        relevant_limits = []
+        for limit in limits:
```

Comment:
might be helpful to uppercase `limits` to `LIMITS` to help signify its an imported const...

Reply:
Done